pkg/ccl/sqltestccl/main_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 The Cockroach Authors.
```

Comment:
i've made a `testccl` package in https://github.com/cockroachdb/cockroach/pull/62238 which ~maps to `pkg/`, so if you did something like `pkg/testccl/sql/...` maybe that's better?